srcflib/plumbing/mysql.py

```diff
@@ -55,23 +55,33 @@ def query(cursor: Cursor, sql: str, *args: Union[str, Tuple[str, ...], Password]
     Run a SQL query against a database cursor, and return whether rows were affected.
     """
     LOG.debug("Query: %r %% %r", sql, args)
-    return bool(cursor.execute(sql, [str(arg) if isinstance(arg, Password) else arg
-                                     for arg in args]))
+    cursor.execute(sql, [str(arg) if isinstance(arg, Password) else arg for arg in args])
+    return bool(cursor.rowcount)
```

Comment:
Would it offer greater utility if we returned the row count, and let the caller decide whether they care about the number or just its nonzeroness?

Reply:
If they want the row count then they're free to look at `cursor.rowcount` -- originally we had several methods looking at whether a query returned results or not so it made sense to shortcut it in the return value, but now there's just one I'm almost inclined to not have it return anything (which would match `pgsql.query()`).

Reply:
Oh, true.  Sorry, I didn't notice that we're passing in `cursor`.